### PR TITLE
i have change the hello world

### PR DESCRIPTION
**Pull request by prabha123456**
hai krishna i have made some changes please check

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <h1>Krishna</h1>
 <p>GOUTHAMI</p>
 
-<p>venkat</p>
+<p>venkat</p> <p>hello world</p>
```
Comment (krishna4721) [inline index.html:7]: i have seen you changes good  i like but some more changes

**Comment by prabha123456** [comment]
done

File: index.html (hunk 1 above)
Comment (krishna4721) [reply index.html:7]: ok thank you

**Comment by krishna4721** [comment]
nice well done...mr

**Pull request merged**